tests/integration/test_local.py

```diff
@@ -0,0 +1,93 @@
+import socket
```

Comment:
Hmm, I'm surprised flake8 didn't complain about this.  Please use pep8 guidelines for grouping imports.